src/commands/distribute/releases/edit.ts

```diff
@@ -70,23 +78,55 @@ export default class EditReleaseCommand extends AppCommand {
       handleHttpError(error, false, "failed to load release details");
     }
 
+    let enabled: boolean = null
+    if(this.state === "enabled" || this.state === "disabled"){
+      enabled = this.state === "enabled"
+    } 
+    const releaseNotes = await this.getReleaseNotesString()
+
     try {
-      debug(`Updating release state to "${state}"`);
+      debug(`Updating release`);
       const httpResponse = await out.progress(
-        `${state === "enabled" ? "Enabling" : "Disabling"} the release...`,
+        `Updating release`,
         clientRequest((cb) =>
-          client.releasesOperations.updateDetails(releaseId, app.ownerName, app.appName, { enabled: state === "enabled" }, cb)
+          client.releasesOperations.updateDetails(releaseId, app.ownerName, app.appName, { enabled: enabled , releaseNotes: releaseNotes }, cb)
```

Comment:
Did you test to make sure this doesn't override the release notes with nothing when no the user provides no release notes?

Sadly the CLI cannot distinguish between empty string parameters and the parameter not being provided. We should still allow users to provide an empty file, and use that to set the release notes to nothing, if the API allows that.

Reply:
@anywherepilot yes we should. Do u have any clear idea how to implement it?

Reply:
Or we should to create 2 separate methods such as 'notes update' and 'notes reset'?

Reply:
First step is to determine what the API does when it receives empty release notes. It either sets the notes to empty, or ignores it. If it sets the nodes to empty: 1) make sure we don't pass empty just because a user doesn't provide the value, 2) make sure we do pass empty when a user explicitly passes in an empty file. If it ignores the value: no changes necessary, current code is fine.